panacea/sgxdb_util.go

```diff
@@ -0,0 +1,83 @@
+package panacea
+
+import (
+	"bytes"
+	"encoding/gob"
+	"github.com/medibloc/panacea-doracle/store/sgxleveldb"
+	log "github.com/sirupsen/logrus"
+	"os"
+	"path/filepath"
+)
+
+func SaveTrustedBlockInfo(info TrustedBlockInfo) error {
+	var buffer bytes.Buffer
+
+	enc := gob.NewEncoder(&buffer)
+
+	err := enc.Encode(info)
+	if err != nil {
+		return err
+	}
+
+	userHomeDir, err := os.UserHomeDir()
+	if err != nil {
+		return err
+	}
+	dbDir := filepath.Join(userHomeDir, ".doracle", "data")
+
+	db, err := sgxleveldb.NewSgxLevelDB("light-client-db", dbDir)
+	if err != nil {
+		return err
+	}
+	err = db.Set([]byte("trustedBlockInfo"), buffer.Bytes())
```

Comment:
Is the key `[]byte("trustedBlockInfo")` the same key used to store and retrieve the trusted block info by light client?

Reply:
Light client does not store `trustedBlockInfo` directly. It just stores `LightBlock` that has signed header and validator set. So, I saved the data with a new key `trustedBlockInfo`.

Reply:
Ah, I got it. Then, we need to update this `trustedBlockInfo` when trusted `LightBlock` is updated as well?

Reply:
I think it's not necessary. The `trustedBlockInfo` is used to initialize and get `LightBlock` from primary. Then, for validation or update, the `Lightblock` is used and `trustedBlockInfo` is not.

Reply:
Understood.

I think that when `register-oracle` cmd is executed, the trusted block info should be set using the `SaveTrustedBlockInfo` function. 
Then, when `get-oracle-key` to be shared the oracle key, a query client is created with the trusted block info using `GetTrustedBlockInfo` function. 
After this initialization of query client (which includes light client), the light client will update and use the `LightBlock` as you said.

Reply:
You're right. Good!

Reply:
If the trusted block info given by operator(person) is only used to initialize the light client, actually we don't have to utilize sgx level DB. We can just seal and store the trusted block info as a file(e.g. .json). wdyt?

At first, I misunderstood that the light client uses and updates the trusted block info, so I said that this util functions are needed.

Reply:
Yes.. I agree with your opinion. There seems to be no reason to use levelDB.

Reply:
OK. Then, let's store it as a file, not using level DB.
You can remove the sgxdb_util.go file.. sorry for the confusion

Reply:
OK! 
No. I didn't even think deeply :).